builder/aggregation/hll_sketch.go

```diff
@@ -0,0 +1,55 @@
+package aggregation
+
+// HLLSketch holds the HLL sketch struct based on
+// Aggregator section in https://druid.apache.org/docs/latest/development/extensions-core/datasketches-hll.html
+type HLLSketch struct {
+	Base
+	FieldName  string `json:"fieldName,omitempty"`
+	LgK        int64  `json:"lgK,omitempty"`
+	TgtHllType string `json:"tgtHllType,omitempty"`
```

Comment:
probably this applies here https://github.com/golang/go/wiki/CodeReviewComments#initialisms